src/__tests__/TopBar.test.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import {TopBar} from '../components/topbar/TopBar'
+import {mount, shallow} from 'enzyme'
+
+describe('TopBar tests', function () {
+  it('TopBar is working', () => {
+    const component = shallow(<TopBar/>)
+    expect(component.find('#topbar-title').text()).toEqual("Hello, I'm the TopBar!")
+  })
+
+  it('TopBar is toggling', () => {
+    let lastToggle = false
+    const component = mount(
+      <TopBar
+        onBurgerClick={(isToggled) => {
```

Comment:
I'd suggest to use a `jest.fn()` and test that it's called with a specific `isToggled` value. 
It's more readable an simple, and allow you to test how many times it's called